chef.gemspec

```diff
@@ -46,7 +46,7 @@ Gem::Specification.new do |s|
   s.add_dependency "ohai", "~> 19.0"
   s.add_dependency "inspec-core", "= 7.0.38.beta"
 
-  s.add_dependency "ffi", ">= 1.15.5"
+  s.add_dependency "ffi", ">= 1.15.5", "< 1.18.0"
```

Comment:
just because?

Reply:
@tpowell-progress look into the ffi pin here

Reply:
You're pinning it it in 4 out of 5 places though...